MapboxMobileEvents/MMEUIApplicationWrapper.h

```diff
@@ -1,14 +1,54 @@
 #import <UIKit/UIKit.h>
 
+NS_ASSUME_NONNULL_BEGIN
+
+// MARK: -
+
 @protocol MMEUIApplicationWrapper <NSObject>
```

Comment:
We will need this to work on macOS as well, `MMApplicationWrapper` is a more future-proof name

Reply:
Although possible, re-naming class/protocol conventions could cause pain for integration of 1.0 Beta branch in the future leading to an overly manual process.

How would you feel about holding off on this until after that is merged. Then more ideal naming/oranizing of files folders could take place such as support for SwiftPM.